DuckDuckGo/Statistics/GeneralPixel.swift

```diff
@@ -627,9 +627,9 @@ enum GeneralPixel: PixelKitEventV2 {
         case .duckPlayerContingencyLearnMoreClicked:
             return "duckplayer_mac_contingency_learn-more-clicked"
         case .dashboardProtectionAllowlistAdd:
-            return "m_mac_mp_wla"
+            return "mp_wla"
         case .dashboardProtectionAllowlistRemove:
-            return "m_mac_mp_wlr"
+            return "mp_wlr"
```

Comment:
@SlayterDev did you test this? I don't think this is going to work as you are firing this as `GeneralPixel`

Reply:
Ok it works now, but a follow up question: `https://improving.duckduckgo.com/t/mp_wla?trigger_origin=primaryScreen&pixelSource=browser-dmg` other platform send app version, what is the spec for this pixel? Is it required?